docs/docusaurus/docs/guides/expectations/components_how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data/_preface.mdx

```diff
@@ -10,3 +10,39 @@ This guide will take you through the process of creating <TechnicalTag tag="expe
 - [Created a Datasource](/docs/guides/connecting_to_your_data/connect_to_data_overview).
 
 </Prerequisites>
+
+## Steps
+
+### 1. Import Great Expectations and get a Data Context
```

Comment:
Why are you using empty headings instead of a simple procedure? This information would be better presented as a procedure. All of the headings make it visually cluttered.

Reply:
The use of headings was to build out the right hand navigation, which is automatically built by Docusaurus based on headers.  I think we should revisit this, update our doc standards, and then standardize the changes as it's own work - revising our current style guidance would be out of scope for this PR.